dev.vencord.Vesktop.metainfo.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.1-->
+  <id>dev.vencord.Vesktop</id>
+  <name>Vesktop</name>
+  <summary>Snappier Discord app/experience with Vencord</summary>
```

Comment:
I didn't mean literal "app/experience", use either one of them. This looks very weird.